src/haz3lschool/Gradescope.re

```diff
@@ -0,0 +1,126 @@
+open Haz3lcore;
```

Comment:
Can you update the INSTRUCTORS.md file in the root directory with instructions for how Gradescope export works

Reply:
Done